### DISCUSSION TITLE
Default state for row - collapsed

### DISCUSSION THREAD

**Discussion by anshibanov** [grafana/grafana#52520] [Dashboards, Panels & Visualizations]
We have many rows at our dasboard.
It would be nice to be able to set the rows's default state to collapsed. So that when the dashboard is loaded or reloaded, it would be collapsed

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
